src/core/Refiner.ml

```diff
@@ -580,6 +558,54 @@ struct
     let+ tp, fam = quantifier tac_base tac_fam univ in
     S.CodeSg (tp, fam)
 
+  let quote_code_sign_hooks (sign : (Ident.user * D.con) list) 
```

Comment:
Can we add a comment / documentation for each of these functions? It would be good to just say briefly what they do.

Reply:
I've added some comments to the new functions, how does it look now?

Reply:
looks good thanks!